code/modules/mob/living/simple_animal/slime/life.dm

```diff
@@ -129,8 +134,27 @@
 	else
 		Tempstun = 0
 
+	if(stat != DEAD)
+		var/co2_percentage =0
+		if(("co2" in environment.gases) && environment.total_moles())
```

Comment:
If you're removing CO2, leaving the turf with flat zero moles, and not calling garbage_collect() afterwards, it's a bug. I'd rather it not fail silently.